### ISSUE TITLE
Pixel Loads Over 10s on Checkout

### ISSUE THREAD

**Issue by digirev** [facebookincubator/facebook-for-woocommerce#214]
Hi there,

I realise there is 30 checkout events that Google recorded, but i can't find it on FB.

The campaign is only on Facebook, so definitely it's Facebook traffic.

When I check the checkout page using Pixel Helper, the InitiateCheckout took more than 10s to load.

How to fix this as I have some campaign running on FB & I feel I lost it.

Thanks.

Regards,
Ruben

**Comment by tahoff** [comment]
Thanks for flagging this @digirev. Can you send your store URL and ensure that you're using the latest version of Facebook for WooCommerce?

Depending on where you looked at your insights, you may see data here: https://www.intern.facebook.com/events_manager/pixel/. Depending on what interface you're looking at to get your insights, it's possible that InitiateCheckout events are happening but from people who didn't see/click on your ads, so the events aren't attributed to the ads you're running. You can learn more about event attribution here: https://www.facebook.com/business/help/458681590974355.

**Comment by digirev** [comment]
Hi, 

Thanks for the reply. 

The website is gizmosified.com. 

The ad doesn't record most of customer's action. When I check on Google,
I can see many events happening. 

I understand that there's a possibility that the traffic is not from
Facebook Ads. However, 99% is definitely from Facebook since it's a new
website & I only advertise through Facebook. 

On 2018-01-16 18:34, Adam Hoff wrote: 

> Thanks for flagging this @digirev [1]. Can you send your store URL and ensure that you're using the latest version of Facebook for WooCommerce? 
> 
> Depending on where you looked at your insights, you may see data here: https://www.intern.facebook.com/events_manager/pixel/ [2]. Depending on what interface you're looking at to get your insights, it's possible that InitiateCheckout events are happening but from people who didn't see/click on your ads, so the events aren't attributed to the ads you're running. You can learn more about event attribution here: https://www.facebook.com/business/help/458681590974355 [3]. 
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub [4], or mute the thread [5].

 

Links:
------
[1] https://github.com/digirev
[2] https://www.intern.facebook.com/events_manager/pixel/
[3] https://www.facebook.com/business/help/458681590974355
[4]
https://github.com/facebookincubator/facebook-for-woocommerce/issues/214#issuecomment-358041570
[5]

**Comment by tahoff** [comment]
@digirev, are you using another pixel plugin on your website? Or did you previously install a pixel using another method? 

Additionally, I'm not able to reproduce the slow pixel fires. I see pixel fire times ~30ms.

**Comment by tahoff** [comment]
Additionally, it's worth noting that Ad blockers can impact the reported number of events from the Facebook Pixel. Does the discrepancy with Google exist for a single event or does the difference exist across a few different events?

**Comment by digirev** [comment]
Hi Adam, 

Thanks for the reply. 

Do you check the product page? 

For example, today I have someone purchased 1 product, but in Facebook
Report it only records ATC without any PUR. 

I can see it on Google Analytics that there's ATC & PUR events happened.


I am afraid it's due to slow speed in pixel. 

At the moment I am using a different pixel plugin (Pixel Caffeine) seems
I am thinking previously it's not recorded anything. 

On 2018-01-17 18:21, Adam Hoff wrote: 

> Additionally, it's worth noting that Ad blockers can impact the reported number of events from the Facebook Pixel. Does the discrepancy with Google exist for a single event or does the difference exist across a few different events? 
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub [1], or mute the thread [2].

 

Links:
------
[1]
https://github.com/facebookincubator/facebook-for-woocommerce/issues/214#issuecomment-358377553
[2]

**Comment by tahoff** [comment]
@digirev, I looked at a variety of pages and all of the pixel fires were between 30-40ms. If you're using a different pixel plugin, then you'll have to message the maintainers. 

It's worth noting that ad blockers can cause differences between what Google reports and what the Facebook pixel reports.

Closing this issue for now, as you've mentioned you're using Pixel Caffeine instead of Facebook for WooCommerce.
